modules/nw-proxy-configure-object.adoc

```diff
@@ -8,9 +8,7 @@
 [id="nw-proxy-configure-object_{context}"]
 = Enabling the cluster-wide proxy
 
-The Proxy object is used to manage the cluster-wide egress proxy. When a cluster is
-installed or upgraded without the proxy configured, a Proxy object is still
-generated but it will have a nil `spec`. For example:
+The Proxy object is used to manage the cluster-wide egress proxy. When a cluster is installed or upgraded without the proxy configured, a Proxy object is still generated but it will have a nil `spec`. For example:
```

Comment:
While you're in here:

"`Proxy` object" or "proxy," AFAIK: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting